0.wave1_data/3.process_features/scripts/extract_sc_interstellar.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Process single cell morphology features for CellProfiler readouts
+
+# ## Import Libraries
+
+# In[1]:
+
+
+import pathlib
+import pandas as pd
+
+import sys
+sys.path.append("../../utils/")
+import sc_extraction_utils as sc_util
+
+from pycytominer.cyto_utils import cells, output
```

Comment:
Were these imports used? If not, consider removing them.

Reply:
Ahhh forgot I left those, I have removed them now!